src/sidebar/util/export-annotations.ts

```diff
@@ -0,0 +1,25 @@
+const formatDate = (date: Date): string => {
+  const year = date.getFullYear();
+  const month = `${date.getMonth() + 1}`.padStart(2, '0');
+  const day = `${date.getDate()}`.padStart(2, '0');
+
+  return `${year}-${month}-${day}`;
+};
+
+type SuggestedFilenameOptions = {
+  group?: string;
```

Comment:
Sorry for the go-around here: perhaps `groupName` to avoid confusion with the `Group` type?